Projects/hCoreUI/Sources/hForm/hRow.swift

```diff
@@ -46,6 +46,7 @@ public struct hRow<Content: View, Accessory: View>: View {
     var accessory: Accessory
     var horizontalPadding: CGFloat = 21
     var verticalPadding: CGFloat = 21
+    var showDivider: Bool = true
```

Comment:
Better switch this to environment property and create modifier to set it to disable but default value should be true.
Later on, you can put that modifier on hSection level and all hRow inside section with inherit that modifier.